packages/dashboards/run-dev.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+##########
+#
+# This script is mean for development of Dashboards and Questions
+# Start Metabase with this script and make the changes you want.
+#
+# After you are done, press ctrl + c and this script automatically
+# saves the database state to MB_DB_SAVE_TO_SQL_FILE
+#
+#########
+
+echo 'OpenCRVS Metabase development environment'
+
+metabase_jar='./metabase.jar'
+
+if [ ! -f "$metabase_jar" ]; then
+  echo "You don't seem to have Metabase installed. Downloading Metabase..."
+  curl https://downloads.metabase.com/v0.45.2.1/metabase.jar --output $metabase_jar
+fi
+
+if [ -f .env ]
+then
+  source .env
+fi
+
+# Variables exported so they can be used in initialize-database.sh
+export OPENCRVS_METABASE_SITE_NAME=${OPENCRVS_METABASE_SITE_NAME:-"OpenCRVS Local"}
+export OPENCRVS_ENVIRONMENT_CONFIGURATION_SQL_FILE=${OPENCRVS_ENVIRONMENT_CONFIGURATION_SQL_FILE:-"$(pwd)/environment-configuration.sql"}
+export MB_DB_FILE=${MB_DB_FILE:-"$(pwd)/data/metabase/metabase.mv.db"}
+export MB_DB_INIT_SQL_FILE=${MB_DB_INIT_SQL_FILE:-"$(pwd)/metabase.init.db.sql"}
+export MB_DB_SAVE_TO_SQL_FILE=${MB_DB_SAVE_TO_SQL_FILE:-"$(pwd)/metabase.init.db.sql"}
```

Comment:
So, could we load from Farajaland in dev and a shared volume in deployment?

Reply:
Good idea, yes! That's definitely something we should do